src/ontology/imports/foodon_product_import.owl

```diff
@@ -33368,6 +33368,52 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
     
 
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004142 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004142">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00002951"/>
+        <obo:IAO_0000115 xml:lang="en">A patty made from rabbit meat.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date xml:lang="en">2021-12-20T15:39:21Z</dc:date>
+        <rdfs:label xml:lang="en">rabbit patty</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004143 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004143">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00004125"/>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date xml:lang="en">2021-12-20T15:36:23Z</dc:date>
+        <rdfs:label xml:lang="en">braided chew</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004144 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004144">
```

Comment:
Definition?

Reply:
I also think that 'braided chew' usually refers to braided bully/pizzle sticks. Couldn't find any other chew in a braided shape other than bully stick. So to define it, maybe we should have a label 'braided pizzle stick' underneath 'pizzle stick'.  Definition: 'A durable pizzle stick made of 3 strips of pizzle that have been cut lengthwise, tightly braided together and then baked at a low temperature'. what do you think?

Reply:
Then 'braided chew' can be a narrow synonym.

Reply:
no need to say now many strips in the braid? or how tightly?  or how low temperature. They might even be dried in the sun?!  Keep it minimal!
"A pizzle stick made of braided strips." perhaps.

Reply:
Right, that's better.